### ISSUE TITLE
Patapon 3 not connecting to server

### ISSUE THREAD

**Issue by boy13epic** [hrydgard/ppsspp#13281]
### What happens?
Connection timeout

### What *should* happen?
Connect to the server

### What hardware, operating system, and PPSSPP version?  On desktop, GPU matters for graphical issues.
iPAD 6 JIT,iOS and latest github build

**Comment by boy13epic** [comment]
Was trying to use fake psn connection

**Comment by hrydgard** [comment]
I think that code is still very early and experimental @ANR2ME  ?

**Comment by ANR2ME** [comment]
Yes, we don't even have any of the infrastructure API implemented LOL how can it works.

**Comment by adenovan** [comment]
psp use old sslv2 and sslv3 some games even did not encrypt the traffic and run it pure through http get and post call.

Even the official psp browser is not much use these days they re not working with many site that use tls to encrypt https call.

Most of the traffic its will be tcp and http
